tests/general/Dos2UnixBearTest.py

```diff
@@ -0,0 +1,11 @@
+from bears.general.Dos2UnixBear import Dos2UnixBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file = """The 50's were swell.
```

Comment:
it would be better to use Python raw strings here, using escape sequences to show which are good and bad.  Otherwise the good and bad look very similar.